sl/CMakeLists.txt

```diff
@@ -104,11 +99,12 @@ if(ENABLE_LLVM)
     # helping scripts for llvm
     configure_file(${PROJECT_SOURCE_DIR}/slscript.in  ${PROJECT_BINARY_DIR}/slllvm    @ONLY)
     configure_file(${PROJECT_SOURCE_DIR}/slllvmv.in   ${PROJECT_BINARY_DIR}/slllvmv   @ONLY)
+    configure_file(${PROJECT_SOURCE_DIR}/sldbg.in     ${PROJECT_BINARY_DIR}/sldbg     @ONLY)
 else()
     # helping scripts for gcc
     configure_file(${PROJECT_SOURCE_DIR}/slscript.in  ${PROJECT_BINARY_DIR}/slgcc     @ONLY)
     configure_file(${PROJECT_SOURCE_DIR}/slgccv.in    ${PROJECT_BINARY_DIR}/slgccv    @ONLY)
-    configure_file(${PROJECT_SOURCE_DIR}/slgdb.in     ${PROJECT_BINARY_DIR}/slgdb     @ONLY)
+    configure_file(${PROJECT_SOURCE_DIR}/sldbg.in     ${PROJECT_BINARY_DIR}/sldbg     @ONLY)
```

Comment:
Why are you renaming `slgdb` to `sldbg`?  It breaks backward compatibility for no real benefit.

Reply:
Because gdb needs turn off 'System Integrity Protection' for debug on mac (I don't want it), so I create script for lldb. gdb+lldb => dbg

Reply:
I am not saying that the script needs to use gdb internally.  I am just saying that there is no point in renaming the script itself.  Please keep the name of the script as it is.  For example when using `git bisect`, nobody wants to use `slgdb` for some revisions while `sldbg` for other revisions.

Reply:
Sorry, I understand - do not change binary names.